src/main/java/com/uber/cadence/client/WorkflowOptions.java

```diff
@@ -356,6 +379,9 @@ public String toString() {
         + ", cronSchedule='"
         + cronSchedule
         + '\''
+        + ", memo='"
+        + memo.toString()
```

Comment:
What if memo is null?
In Java no need to call .toString when summing up a string. Just change to "memo="  + memo +

Reply:
Updated, thanks